stdlib/ballerina-http/src/main/ballerina/http/http_retry_client.bal

```diff
@@ -21,7 +21,7 @@ public type RetryClient object {
         HttpClient httpClient;
     }
 
-    new (serviceUri, config, retry, httpClient) {
+    public new (serviceUri, config, retry, httpClient) {
         self.serviceUri = serviceUri;
```

Comment:
You don't have to do this. The whole point is that is your object field names and constructor args with no type names match, then you don't have to write, self.foo = foo. They get auto-assigned.

Reply:
Fixed.